src/app/youtube/models/search-item.model.ts

```diff
@@ -0,0 +1,42 @@
+interface IThumbnail {
+    "url": string,
+    "width": number,
+    "height": number
+}
+
+interface IStatistics {
+    "viewCount": string,
+    "likeCount": string,
+    "dislikeCount": string,
+    "favoriteCount": string,
+    "commentCount": string
+}
+
+export interface SearchItem {
+    "kind": string,
+    "etag": string,
+    "id": string,
+    "snippet": {
+        "publishedAt": string,
```

Comment:
It's much better now, but it could be better.
Please refactor it . Each `{` means new object.

Reply:
I refactored it